code/src/components/CompleteAll.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+// Reducer
+import todo from 'reducers/todo'
+
+import styled from 'styled-components/macro'
+import { AddStartIcon, StartBtn } from 'GlobalStyle'
+import allDoneBtn from '../assets/allDone_icon.png'
+
+export const CompleteAll = () => {
+  const dispatch = useDispatch()
+
+  const onCompleteAll = () => {
+    dispatch(todo.actions.completeAll())
+  }
+
+  return (
+    <AllDoneBtn onClick={onCompleteAll}>
+      <AllDoneIcon src={allDoneBtn} alt="All todos done" />
+    </AllDoneBtn>
+  )
+}
+
+// Styled Components
+
+const AllDoneBtn = styled(StartBtn)`
+  
+`;
+
+const AllDoneIcon = styled(AddStartIcon)`
+  padding: 1em 0 1em 0.4em;
+  border: none;
+  width: 70px;
+  
+
```

Comment:
Nice that you added a complete all component, very cool that you had time to add that. Im IMPRESSED 🥇